robot_smach_states/src/robot_smach_states/navigation/navigate_to_grasp.py

```diff
@@ -38,7 +38,7 @@ def generateConstraint(self):
         if arm == self.robot.arms['left']:
             angle_offset = math.atan2(-self.robot.grasp_offset.y, self.robot.grasp_offset.x)
         elif arm == self.robot.arms['right']:
-            angle_offset = math.atan2(self.robot.grasp_offset.y, self.robot.grasp_offset.x)
+            angle_offset = math.atan2(-self.robot.grasp_offset.y, self.robot.grasp_offset.x)
```

Comment:
This is a big difference between Amigo and Hero. I suppose this should be ```arm.grasp_offset.y``` where the left arm is the same as the negative offset of the right arm, which should be defined in Amigo and Hero description.

Reply:
We can solve this indeed by defining arm-specific grasp_offsets, so we also don't have to check for left/right. 
Merging like it is now will break for AMIGO, not changing it will break for HERO. 
I think the easiest solution for now, given that generalizing the arm indices is a separate PR, is to add a check for AMIGO and HERO and removing that check when the arms are generalized for left and right. 
It's not pretty but workable and very clear that is should be cleaned up.